docs/configuration.rst

```diff
@@ -0,0 +1,92 @@
+|                                          | Example: /media/ephemeral\ **0** to /media/ephemeral\ **4**            |                   |                  |               |
++------------------------------------------+------------------------------------------------------------------------+-------------------+------------------+---------------+
+| hadoop.cache.data.dirsuffix              | The name of the subdirectory to be used for storing cache files.       | string            | /fcache/         | C & S         |
+|                                          | Example: /media/ephemeral0\ **/fcache/**                               |                   |                  |               |
++------------------------------------------+------------------------------------------------------------------------+-------------------+------------------+---------------+
+| hadoop.cache.data.expiration.after-write | The time files will be kept in cache prior to eviction.                | integer (ms)      | MAX_VALUE        | S             |
++------------------------------------------+------------------------------------------------------------------------+-------------------+------------------+---------------+
+| hadoop.cache.data.fullness.percentage    | The percentage of the disk space that will be filled with cached data  | integer (%)       | 80               | S             |
+|                                          | before cached files will start being evicted.                          |                   |                  |               |
++------------------------------------------+------------------------------------------------------------------------+-------------------+------------------+---------------+
+| hadoop.cache.data.strict.mode            | Propagate exceptions if there is an error while caching data if true;  | boolean           | false            | C             |
+|                                          | otherwise fall back on reading data directly from remote file system   |                   |                  |               |
++------------------------------------------+------------------------------------------------------------------------+-------------------+------------------+---------------+
+| rubix.enable.file.staleness-check        | Check file metadata for updates from remote filesystem when true.      | boolean           | true             | S             |
++------------------------------------------+------------------------------------------------------------------------+-------------------+------------------+---------------+
+| rubix.stale.fileinfo.expiry.period       | (**rubix.enable.file.staleness-check** must be enabled)                |                   |                  |               |
+|                                          | The time file metadata will be cached before expiring.                 | integer (s)       | 36000            | S             |
```

Comment:
**rubix.enable.file.staleness-check** must be set to false. The time file metadata will be cached before it will be fetched again from object store